foundry/applications/hotbar.d.ts

```diff
@@ -1 +1,146 @@
-declare class Hotbar extends Application {}
+/**
+ * The global action bar displayed at the bottom of the game view.
+ * The Hotbar is a UI element at the bottom of the screen which contains Macros as interactive buttons.
+ * The Hotbar supports 5 pages of global macros which can be dragged and dropped to organize as you wish.
+ *
+ * Left clicking a Macro button triggers its effect.
+ * Right clicking the button displays a context menu of Macro options.
+ * The number keys 1 through 0 activate numbered hotbar slots.
+ * Pressing the delete key while hovering over a Macro will remove it from the bar.
+ *
+ * @see {@link Macros}
+ * @see {@link Macro}
+ */
+
+declare class Hotbar extends Application {
+  constructor(options: Application.Options);
```

Comment:
Just noticed, this should probably be
```ts
   constructor(options?: Partial<Application.Options>);
 ```
 as that is also how it is typed in `Application`.